src/main/java/se/sundsvall/webmessagecollector/integration/opene/OpenEMapper.java

```diff
@@ -20,51 +23,55 @@
 
 @Component
 class OpenEMapper {
-    private static final Logger LOG = LoggerFactory.getLogger(OpenEMapper.class);
-    
-    private static final DateTimeFormatter formatter = new DateTimeFormatterBuilder()
-        .appendPattern("yyyy-MM-dd")
-        .optionalStart()
-        .appendPattern(" HH:mm")
-        .optionalEnd()
-        .parseDefaulting(ChronoField.HOUR_OF_DAY, 0)
-        .parseDefaulting(ChronoField.MINUTE_OF_HOUR, 0)
-        .toFormatter();
-    
-    List<MessageEntity> mapMessages(byte[] errands, String familyId) {
-        
-        if (errands == null) {
-            return List.of();
-        }
-        
-        var xmlString = new String(errands, StandardCharsets.ISO_8859_1);
-        try {
-            return new XmlMapper()
-                .readValue(xmlString, Messages.class)
-                .getExternalMessages().stream()
-                .filter(externalMessage -> !externalMessage.isPostedByManager())
-                .map(externalMessage -> {
-                    var entity = MessageEntity.builder()
-                        .withFamilyId(familyId)
-                        .withDirection(externalMessage.isPostedByManager() ? Direction.OUTBOUND : Direction.INBOUND)
-                        .withMessageId(String.valueOf(externalMessage.getMessageID()))
-                        .withExternalCaseId(String.valueOf(externalMessage.getFlowInstanceID()))
-                        .withMessage(externalMessage.getMessage())
-                        .withSent(LocalDateTime.parse(externalMessage.getAdded(), formatter));
-                    if (externalMessage.getPoster() != null) {
-                        entity.withEmail(externalMessage.getPoster().getEmail())
-                            .withFirstName(externalMessage.getPoster().getFirstname())
-                            .withLastName(externalMessage.getPoster().getLastname())
-                            .withUsername(externalMessage.getPoster().getUsername())
-                            .withUserId(String.valueOf(externalMessage.getPoster().getUserID()));
-                    }
-                    return entity.build();
-                })
-                
-                .toList();
-        } catch (JsonProcessingException e) {
-            LOG.info("Something went wrong parsing messages", e);
-            return List.of();
-        }
-    }
+
+	private static final Logger LOG = LoggerFactory.getLogger(OpenEMapper.class);
+
+	private static final DateTimeFormatter formatter = new DateTimeFormatterBuilder()
+		.appendPattern("yyyy-MM-dd")
+		.optionalStart()
+		.appendPattern(" HH:mm")
+		.optionalEnd()
+		.parseDefaulting(ChronoField.HOUR_OF_DAY, 0)
+		.parseDefaulting(ChronoField.MINUTE_OF_HOUR, 0)
+		.toFormatter();
+
+	List<MessageEntity> mapMessages(final byte[] errands, final String familyId) {
+
+		if (errands == null) {
+			return List.of();
```

Comment:
You have added a statical import to: java.util.Collections.emptyList
Maybe change to `emptyList()` here?

(`emptyList()` is more intuitive than `List.of()` for this purpose i.m.h.o.)